pytest_testrail/plugin.py

```diff
@@ -365,3 +378,18 @@ def get_available_testruns(self, plan_id):
                     if not run['is_completed']:
                         testruns_list.append(run['id'])
         return testruns_list
+
+    def get_tests(self, run_id):
+        """
+        :return: the list of tests containing in a testrun.
+
+        """
+        response = self.client.send_get(
+            GET_TESTS_URL.format(run_id),
+            cert_check=self.cert_check
+        )
+        error = self.client.get_error(response)
```

Comment:
```
    if  self.client.get_error(response):
        print('[{}] Failed to get tests: "{}"'.format(TESTRAIL_PREFIX, error))
        return
return response
```
Would that work and be a bit cleaner ?

Reply:
Ok, I use implicit behavior of Python functions that return always `None` if no explicit `return` occurred